app/src/androidTest/java/com/mutualmobile/barricade/sample/BarricadeActivityTest.java

```diff
@@ -74,7 +78,7 @@
     Espresso.pressBack();
```

Comment:
This is not required, Espresso restarts the activity every time before running a test, remove from `verifyResetDialogIsDisplayed()` too